web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -642,11 +642,6 @@ GROUP BY 1`;
       ];
     } else {
       return goToActions.concat([
-        {
-          icon: IconNames.GANTT_CHART,
-          title: 'Go to tasks',
-          onAction: () => goToTask(datasource),
-        },
```

Comment:
why did you remove this?

Reply:
I noticed this was duplicated, if you look carefully there will be two "Go to tasks" under each datasource(unless they are disabled).

Reply:
oh. Great catch!